OtripleS.Web.Api/Brokers/Storage/StorageBroker.cs

```diff
@@ -3,13 +3,14 @@
 // FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
 // ---------------------------------------------------------------
 
-using EFxceptions;
+using EFxceptions.Identity;
 using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.Configuration;
+using OtripleS.Web.Api.Models.Users;
 
 namespace OtripleS.Web.Api.Brokers.Storage
 {
-    public partial class StorageBroker : EFxceptionsContext, IStorageBroker
+    public partial class StorageBroker : EFxceptionsIdentityContext<User>, IStorageBroker
```

Comment:
You should probably do this as follows:
```chsarp
public partial class StorageBroker : EFxceptionsIdentityContext<User, Role, Guid>, IStorageBroker ...

```

Here's your Role Model: 

```csharp
using System;
using Microsoft.AspNetCore.Identity;

namespace Whatever ...
{
    public class Role : IdentityRole<Guid> { }
}

```

The Guid will just be for the unique identifier for the user

Reply:
@hassanhabib resolved, many thanks